sentry/src/main/java/io/sentry/instrumentation/file/FileIOSpanManager.java

```diff
@@ -30,10 +34,23 @@ final class FileIOSpanManager {
   FileIOSpanManager(
       final @Nullable ISpan currentSpan,
       final @Nullable File file,
-      final boolean isSendDefaultPii) {
+      final @NotNull SentryOptions options) {
+    this(
+        currentSpan,
+        file,
+        options,
+        new SentryStackTraceFactory(options.getInAppExcludes(), options.getInAppExcludes()));
```

Comment:
`h`: 
```suggestion
        new SentryStackTraceFactory(options.getInAppExcludes(), options.getInAppIncludes()));
```

Reply:
lol, nice catch! will add a test for this